### PR TITLE
No issue, scrum discussion

### PR DESCRIPTION
**Pull request by swopebe**
Adding info to release note about support.

### CODE REVIEW

File: release_notes/release_notes.adoc [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+*Note:* The 2.2 and earlier versions of {product-title-short} are _removed_ from service, and are no longer supported. Documentation for versions 2.2 and earlier is not updated. The documentation might remain available, but is deprecated without any Errata or other updates available. 
 
 * xref:../release_notes/whats_new.adoc#whats-new[What's new in {product-title}]
 * xref:../release_notes/errata.adoc#errata-updates[Errata updates]
 * xref:../release_notes/known_issues.adoc#known-issues[Known issues and limitations]
 * xref:../release_notes/deprecate_remove.adoc#deprecations-removals[Deprecations and removals]
 * xref:../release_notes/gdpr_readiness.adoc#red-hat-advanced-cluster-management-for-kubernetes-platform-considerations-for-gdpr-readiness[{product-title} considerations for GDPR readiness]
 * xref:../release_notes/fips_readiness.adoc#fips-readiness[FIPS readiness]
+
+To report issues with one of the currently supported releases or its documentation, open a https://bugzilla.redhat.com/enter_bug.cgi?product=Red%20Hat%20Advanced%20Cluster%20Management%20for%20Kubernetes[support ticket].
```
Comment (cadawson) [inline release_notes/release_notes.adoc]: The original link only allowed me to complete an issue against the Customer Portal. I hope that is not a permissions issue that I don't have, but others will also have problems if it is. I replaced with a link to BZ. We might have to change it to Jira at some point. 

I am asking Josh Carter if he knows about any problems with that.

Any thoughts about putting it in a level 2 heading titled "Support" or "Contacting support"?

**Review by cadawson — CHANGES_REQUESTED** [review]
Updated with comments. Let me know if you have questions. Thank you for working on this!

File: release_notes/release_notes.adoc (hunk 1 above)
Comment (swopebe) [reply release_notes/release_notes.adoc]: @cadawson the link was to the support docs, we don't want to link just to BZ when there are other avenues... that is why I chose that link. I recommend we link them to the higher support authority. We would not put contacting support here, but that can go in the Troubleshooting spot.

Comment (swopebe) [reply release_notes/release_notes.adoc]: I will change this to the higher support contact.

Comment (swopebe) [reply release_notes/release_notes.adoc]: We don't use possessives in tech doc as a rule.

Comment (swopebe) [reply release_notes/release_notes.adoc]: Please see changes...we don't want to bypass all questions to just a BZ...the user can get to the case open tab from that spot but also has access to other documentation that may help them first.

**Review by cadawson — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**